bitnami/prometheus-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.36.0
 description: The Prometheus Operator for Kubernetes provides easy monitoring definitions for Kubernetes services and deployment and management of Prometheus instances.
 name: prometheus-operator
-version: 0.12.6
+version: 0.12.7
```

Comment:
As you are adding a new feature this should be a minor change:
```diff
-version: 0.12.6
+version: 0.13.0
```

Reply:
ok, bumped to 0.13.0